ocrd_monitor/flask/app/workflow.py

```diff
@@ -8,7 +9,13 @@
 @bp.route('/workflows/detail', methods=['GET'])
 def detail():
     path = request.args.get('path')
-    content = ''
-    with open(path, 'r') as workflow:
-        content += workflow.read()
-    return render_template('workflows/detail.html', workflow=content)
+    path = path.lstrip('/')
+    path = os.path.join(current_app.config["BASEDIR"], path)

```

Comment:
Not sure this is correct. We currently have `/data/3/ocr-workflow.sh` in there (unless the Kitodo project had some other file assigned and copied). Stripping the leading slash will break this. Also, it used to work so far. 

So I agree we should add a 404 case if the file is simply not there. But absolute paths should stay absolute paths.